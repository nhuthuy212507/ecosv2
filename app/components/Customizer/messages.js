/*
 * Customizer Messages
 *
 * This contains all the text for the Customizer component.
 */

import { defineMessages } from 'react-intl';

export const scope = 'app.components.Customizer';

export default defineMessages({
  header: {
    id: `${scope}.header`,
    defaultMessage: 'This is the Customizer component!',
  },
});
