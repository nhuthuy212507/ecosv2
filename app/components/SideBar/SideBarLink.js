import styled, { css } from 'styled-components';

export default styled.li`
  height: 36px;
  width: 240px;
  transition: all 0.3s;
  position: relative;
  cursor: pointer;
  display: flex;
  padding: 11px 12px;
  overflow: hidden;
  background: transparent;
  border: none;
  color: ${props => props.theme.colorText};
  &:before {
    content: '';
    position: absolute;
    left: 0;
    top: 0;
    height: 100%;
    width: 2px;
    background: var(--color-accent);
    opacity: 0;
    transition: all 0.3s;
  }

  p {
    position: absolute;
    left: 35px;
    width: 180px;
    transition: left 0.3s;
    top: 50%;
    transform: translateY(-50%);
  }

  &:hover {
    text-decoration: none;
    background-color: ${props => props.theme.colorHover};

    &:before {
      opacity: 1;
    }
  }

  ${({ collapse }) =>
    collapse &&
    css`
      @media screen and (min-width: 576px) {
        overflow: hidden;
        width: 40px;
        background-color: transparent;

        p {
          position: absolute;
          left: 55px;
          width: 160px;
        }

        &:hover {
          width: 240px;
          background: ${props => props.theme.colorHover};
        }
      }
    `}
`;
