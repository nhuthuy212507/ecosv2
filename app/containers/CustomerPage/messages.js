/*
 * CustomerPage Messages
 *
 * This contains all the text for the CustomerPage container.
 */

import { defineMessages } from 'react-intl';

export const scope = 'app.containers.CustomerPage';

export default defineMessages({
  titleListPage: {
    id: `${scope}.titleListPage`,
    defaultMessage: 'List Customer',
  },
  titleCreatePage: {
    id: `${scope}.titleCreatePage`,
    defaultMessage: 'Create Customer',
  },
  titleUpdatePage: {
    id: `${scope}.titleUpdatePage`,
    defaultMessage: 'Update Customer',
  },
  name: {
    id: `${scope}.name`,
    defaultMessage: 'Name',
  },
  customerNo: {
    id: `${scope}.customerNo`,
    defaultMessage: 'Customer No',
  },
  typeCustomer: {
    id: `${scope}.typeCustomer`,
    defaultMessage: 'Type',
  },
  personal: {
    id: `${scope}.personal`,
    defaultMessage: 'Personal',
  },
  business: {
    id: `${scope}.business`,
    defaultMessage: 'Business',
  },
  billingAddress: {
    id: `${scope}.billingAddress`,
    defaultMessage: 'Billing Address',
  },
  phone: {
    id: `${scope}.phone`,
    defaultMessage: 'Phone',
  },
  email: {
    id: `${scope}.email`,
    defaultMessage: 'Email',
  },
  notificationEmail: {
    id: `${scope}.notificationEmail`,
    defaultMessage: 'Notification email',
  },
  taxNo: {
    id: `${scope}.taxNo`,
    defaultMessage: 'Tax No',
  },
  status: {
    id: `${scope}.status`,
    defaultMessage: 'Status',
  },
  active: {
    id: `${scope}.active`,
    defaultMessage: 'Active',
  },
  inActive: {
    id: `${scope}.inActive`,
    defaultMessage: 'Inactive',
  },
  customerRef: {
    id: `${scope}.customerRef`,
    defaultMessage: 'Customer Reference',
  },
  createdBy: {
    id: `${scope}.createdBy`,
    defaultMessage: 'Created By',
  },
  paymentMethod: {
    id: `${scope}.paymentMethod`,
    defaultMessage: 'Payment Method',
  },
  cash: {
    id: `${scope}.cash`,
    defaultMessage: 'Cash',
  },
  netOff: {
    id: `${scope}.netOff`,
    defaultMessage: 'NetOff',
  },
  bankTransfer: {
    id: `${scope}.bankTransfer`,
    defaultMessage: 'Bank Transfer',
  },
  description: {
    id: `${scope}.description`,
    defaultMessage: 'Description',
  },
  website: {
    id: `${scope}.website`,
    defaultMessage: 'Website',
  },
  fax: {
    id: `${scope}.fax`,
    defaultMessage: 'Fax',
  },
  city: {
    id: `${scope}.city`,
    defaultMessage: 'City',
  },
  district: {
    id: `${scope}.district`,
    defaultMessage: 'District',
  },
  ward: {
    id: `${scope}.ward`,
    defaultMessage: 'Ward',
  },
  address: {
    id: `${scope}.address`,
    defaultMessage: 'Address',
  },
  action: {
    id: `${scope}.action`,
    defaultMessage: 'Action',
  },
  confirmMessage: {
    id: `${scope}.confirmMessage`,
    defaultMessage: 'Are you sure',
  },
  add: {
    id: `${scope}.add`,
    defaultMessage: 'Add',
  },
  create: {
    id: `${scope}.create`,
    defaultMessage: 'Create',
  },
  search: {
    id: `${scope}.search`,
    defaultMessage: 'Search',
  },
  clear: {
    id: `${scope}.clear`,
    defaultMessage: 'Clear',
  },
  cancel: {
    id: `${scope}.cancel`,
    defaultMessage: 'Cancel',
  },
  delete: {
    id: `${scope}.delete`,
    defaultMessage: 'Delete',
  },
  save: {
    id: `${scope}.save`,
    defaultMessage: 'Save',
  },
  yes: {
    id: `${scope}.yes`,
    defaultMessage: 'Yes',
  },
  no: {
    id: `${scope}.no`,
    defaultMessage: 'No',
  },
  collapse: {
    id: `${scope}.collapse`,
    defaultMessage: 'Collapse',
  },
  nameRequired: {
    id: `${scope}.nameRequired`,
    defaultMessage: 'Nam is required',
  },
  nameExists: {
    id: `${scope}.nameExists`,
    defaultMessage: 'Name already exists',
  },
  phoneRequired: {
    id: `${scope}.phoneRequired`,
    defaultMessage: 'Phone is required',
  },
  typeRequied: {
    id: `${scope}.typeRequied`,
    defaultMessage: 'Type customer is required',
  },
  customerNoRequired: {
    id: `${scope}.customerNoRequired`,
    defaultMessage: 'Customer No is required',
  },
  emailRequired: {
    id: `${scope}.emailRequired`,
    defaultMessage: 'Email is required',
  },
  emailInvalid: {
    id: `${scope}.emailInvalid`,
    defaultMessage: 'Invalid email',
  },
  taxCodeRequired: {
    id: `${scope}.taxCodeRequired`,
    defaultMessage: 'Tax No is required',
  },
  notiEmailRequired: {
    id: `${scope}.notiEmailRequired`,
    defaultMessage: 'Notification email is required',
  },
  cityRequired: {
    id: `${scope}.cityRequired`,
    defaultMessage: 'City is required',
  },
  districtRequired: {
    id: `${scope}.districtRequired`,
    defaultMessage: 'District is required',
  },
  wardRequired: {
    id: `${scope}.wardRequired`,
    defaultMessage: 'Ward is required',
  },
  addressRequired: {
    id: `${scope}.addressRequired`,
    defaultMessage: 'Address is required',
  },
  badRequest: {
    id: `${scope}.badRequest`,
    defaultMessage: 'Bad Request',
  },
  warehouseAddressExists: {
    id: `${scope}.warehouseAddressExists`,
    defaultMessage: 'Warehouse Address already exists',
  },
  addCustomerSuccess: {
    id: `${scope}.addCustomerSuccess`,
    defaultMessage: 'Create successful customer',
  },
  editCustomerSuccess: {
    id: `${scope}.editCustomerSuccess`,
    defaultMessage: 'Update successful customer',
  },
  addWarehouseAddressSuccess: {
    id: `${scope}.addWarehouseAddressSuccess`,
    defaultMessage: 'Add warehouse address success',
  },
  editWarehouseAddressSuccess: {
    id: `${scope}.editWarehouseAddressSuccess`,
    defaultMessage: 'Update warehouse address success',
  },
  setDefaulAddresstSuccess: {
    id: `${scope}.setDefaulAddresstSuccess`,
    defaultMessage: 'Set default address success',
  },
});
