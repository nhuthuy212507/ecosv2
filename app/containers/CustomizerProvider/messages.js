/*
 * CustomizerProvider Messages
 *
 * This contains all the text for the CustomizerProvider component.
 */
import { defineMessages } from 'react-intl';

export const scope = 'app.containers.CustomizerProvider';

export default defineMessages({
  title: {
    id: `${scope}.title`,
    defaultMessage: 'Theme Customizer',
  },
  darkTheme: {
    id: `${scope}.darkTheme`,
    defaultMessage: 'Dark Theme',
  },
  collapsedMenu: {
    id: `${scope}.collapsedMenu`,
    defaultMessage: 'Collapsed Menu',
  },
});
