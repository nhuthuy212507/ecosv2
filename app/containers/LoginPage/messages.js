/*
 * LoginPage Messages
 *
 * This contains all the text for the LoginPage container.
 */

import { defineMessages } from 'react-intl';

export const scope = 'app.containers.LoginPage';

export default defineMessages({
  helmet: {
    id: `${scope}.helmet`,
    defaultMessage: 'Login Page',
  },
  welcome: {
    id: `${scope}.welcome`,
    defaultMessage: 'Welcome to',
  },
  subHead: {
    id: `${scope}.subHead`,
    defaultMessage: 'Please use your credentials to login',
  },
  rememberMe: {
    id: `${scope}.rememberMe`,
    defaultMessage: 'Remember Me',
  },
  login: {
    id: `${scope}.login`,
    defaultMessage: 'Log in',
  },
  usernameRequired: {
    id: `${scope}.usernameRequired`,
    defaultMessage: 'Username is required',
  },
  passwordRequired: {
    id: `${scope}.passwordRequired`,
    defaultMessage: 'Password is required',
  },
  incorrectInfo: {
    id: `${scope}.incorrectInfo`,
    defaultMessage: 'Username or password incorrect',
  },
  badRequest: {
    id: `${scope}.badRequest`,
    defaultMessage: 'Opps! This is bad request',
  },
  error401: {
    id: `${scope}.error401`,
    defaultMessage: 'Please login to continue',
  },
});
