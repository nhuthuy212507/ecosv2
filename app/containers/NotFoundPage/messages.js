/*
 * NotFoundPage Messages
 *
 * This contains all the text for the NotFoundPage container.
 */
import { defineMessages } from 'react-intl';

export const scope = 'app.containers.NotFoundPage';

export default defineMessages({
  goHome: {
    id: `${scope}.goHome`,
    defaultMessage: 'Go Home',
  },
  subTitle: {
    id: `${scope}.subTitle`,
    defaultMessage: 'Sorry, the page you visited does not exist.',
  },
});
