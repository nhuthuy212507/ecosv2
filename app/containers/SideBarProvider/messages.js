/*
 * SideBarProvider Messages
 *
 * This contains all the text for the SideBarProvider container.
 */

import { defineMessages } from 'react-intl';

export const scope = 'app.containers.SideBarProvider';

export default defineMessages({
  dashboard: {
    id: `${scope}.dashboard`,
    defaultMessage: 'Dashboard',
  },
  customer: {
    id: `${scope}.customer`,
    defaultMessage: 'Customer',
  },
});
