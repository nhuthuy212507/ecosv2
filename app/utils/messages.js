/*
 * Util Notification Messages
 *
 * This contains all the text for the Util notification
 */

import { defineMessages } from 'react-intl';

export const scope = 'app.utils.Notification';

export default defineMessages({
  error405: {
    id: `${scope}.error405`,
    defaultMessage: 'Method not allow',
  },
  error500: {
    id: `${scope}.error500`,
    defaultMessage: 'Opps! Server is wrong',
  },
});
