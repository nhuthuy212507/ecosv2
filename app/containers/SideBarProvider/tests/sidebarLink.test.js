/**
 * Test the HomePage
 */

import React from 'react';
import { render } from 'react-testing-library';
import { Provider } from 'react-redux';
import { browserHistory, BrowserRouter } from 'react-router-dom';
import { FormattedMessage, IntlProvider } from 'react-intl';

import messages from '../messages';
import SideBarLink from '../sidebarLink';
import configureStore from '../../../configureStore';

describe('<sidebarLink />', () => {
  let store;

  beforeAll(() => {
    store = configureStore({}, browserHistory);
  });

  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <Provider store={store}>
        <IntlProvider locale="en">
          <BrowserRouter>
            <SideBarLink
              collapse={false}
              title={<FormattedMessage {...messages.dashboard} />}
              route="/test"
            />
          </BrowserRouter>
        </IntlProvider>
      </Provider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
