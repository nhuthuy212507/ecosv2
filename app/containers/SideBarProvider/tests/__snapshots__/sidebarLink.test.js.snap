// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<sidebarLink /> should render and match the snapshot 1`] = `
<a
  class="sidebarLink__StyledLink-gs9h8q-0 jnfaer"
  href="/test"
>
  <li
    class="SideBarLink-u8n3ub-0 enzljv"
  >
    
    <p
      class="SideBarLinkTitle-cse4j8-0 ChVqR"
    >
      <span>
        Dashboard
      </span>
    </p>
  </li>
</a>
`;
